valkey.conf

```diff
@@ -2319,3 +2319,9 @@ jemalloc-bg-thread yes
 # to suppress
 #
 # ignore-warnings ARM64-COW-BUG
+
+# Inform Valkey of the availability zone if running in a cloud environment.  Currently
+# this is only exposed via the info command for clients to use, but in the future we
+# we may also use this when making decisions for replication.
+#
+# availability-zone "us-east-1a"
```

Comment:
```suggestion
# availability-zone "us-east-1a"
```
This seems AWS'y. Can we have a more general term like "network-zone" ? @PingXie @soloestoy @hwware Would be good to get your input on this.

Reply:
Done

Reply:
Ah I thought it was the reference to us-east-1 :) 
Availability zone is ideal because then if we want to migrate to Valkey we can do it in place instead of a bunch of workarounds and I think its a fairly generic term.

Reply:
+1 for availability-zone. It's a known term. https://en.wikipedia.org/wiki/Availability_zone

Kubernetes uses the term failure zones and regions, but they added "also known as availability zones". https://kubernetes.io/docs/setup/best-practices/multiple-zones/

Reply:
Well `us-east-1` looks familiar too :)

I am biased towards `availability-zone`, tbh, but I was trying to see through the lens of self-managed on-prem users. I think @zuiderkwast is the least biased among us so if he is happy, I am in too :-)

Reply:
i.m ok with the config name `availability-zone`, and the example name "zone-name" is good, avoid too "clould'y" ;)

Reply:
Because I force pushed its not showing here but I changed the example to:
`# availability-zone "zone-name"`

Even in an on-prem environment if you create isolated zones AZ is a good name, but I think it's less common outside the cloud.

Reply:
I'm happy if others are happy, this looks resolved.